### PR TITLE
ft: add tests for bucketMatch false

### PR DESCRIPTION
**Pull request by dora-korpar**
Adds tests for the get and delete APIs when bucketMatch = false, which was handled but not tested.

### CODE REVIEW

**Comment by dora-korpar** [comment]
@ironman-machine try

**Comment by dora-korpar** [comment]
@ironman-machine try

**Comment by dora-korpar** [comment]
E2E passed here: http://ci.ironmann.io/gh/scality/Integration/15084

File: tests/functional/aws-node-sdk/test/multipleBackend/get/get.js [hunk 1]
```diff
@@ -293,5 +294,23 @@ describe('Multiple backend get object', function testSuite() {
...
+                Metadata: { 'scal-location-constraint': awsLocationMismatch } },
+                err => {
+                    assert.equal(err, null, `Err putting object: ${err}`);
+                    done();
+                });
+            });
+
+            it('should get an object from AWS', done => {
+                s3.getObject({ Bucket: bucket, Key: mismatchObject }, err => {
+                    assert.equal(err, null, `Error getting object: ${err}`);
```
Comment (electrachong) [inline tests/functional/aws-node-sdk/test/multipleBackend/get/get.js]: you won't assert the data body?

File: tests/functional/aws-node-sdk/test/multipleBackend/get/get.js [hunk 2]
```diff
@@ -293,5 +294,25 @@ describe('Multiple backend get object', function testSuite() {
                 });
             });
         });
+
+        describeSkipIfNotMultiple.only('with bucketMatch set to false', () => {
```
Comment (philipyoo) [inline tests/functional/aws-node-sdk/test/multipleBackend/get/get.js]: `.only` 😨

Comment (dora-korpar) [reply tests/functional/aws-node-sdk/test/multipleBackend/get/get.js]: omg thanks, I just pushed that :embarrassed:

**Comment by dora-korpar** [comment]
@ironman-machine try

**Comment by nicolas2bert** [comment]
@ironman-machine r+

**Comment by LaurenSpiegel** [comment]
@ironman-machine r+

**Pull request merged**